include/bluetooth/audio/aics.h

```diff
@@ -0,0 +1,450 @@
+#define ZEPHYR_INCLUDE_BLUETOOTH_SERVICES_AICS_H_
+
+/**
+ * @brief Audio Input Control Service (AICS)
+ *
+ * @defgroup bt_gatt_aics Audio Input Control Service (AICS)
+ *
+ * @ingroup bluetooth
+ * @{
+ *
+ * The Audio Input Control Service is a secondary service, and as such should not be used own its
+ * own, but rather in the context of another (primary) service.
+ *
+ * This API implements both the server and client functionality.
+ * Note that the API abstracts away the change counter in the audio input control state and will
+ * automatically handle any changes to that. If out of date, the client implementation will
+ * autonomously read the change counter value when executing a write request.
+ *
+ * [Experimental] Users should note that the APIs can change as a part of ongoing development.
+ */
+
+#include <zephyr/types.h>
+#include <bluetooth/bluetooth.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/** Audio Input Control Service mute states */
+#define BT_AICS_STATE_UNMUTED                      0x00
+#define BT_AICS_STATE_MUTED                        0x01
+#define BT_AICS_STATE_MUTE_DISABLED                0x02
+
+/** Audio Input Control Service input modes */
+#define BT_AICS_MODE_MANUAL_ONLY                   0x00
+#define BT_AICS_MODE_AUTO_ONLY                     0x01
+#define BT_AICS_MODE_MANUAL                        0x02
+#define BT_AICS_MODE_AUTO                          0x03
+
+/** Audio Input Control Service input types */
+#define BT_AICS_INPUT_TYPE_LOCAL                   0x00
+#define BT_AICS_INPUT_TYPE_ISO                     0x01
+#define BT_AICS_INPUT_TYPE_ANALOG                  0x02
+#define BT_AICS_INPUT_TYPE_DIGITAL                 0x03
+#define BT_AICS_INPUT_TYPE_RADIO                   0x04
+#define BT_AICS_INPUT_TYPE_PHYS_MEDIA              0x05
+#define BT_AICS_INPUT_TYPE_NETWORK                 0x06
+#define BT_AICS_INPUT_TYPE_OTHER                   0x255
+
+/** Audio Input Control Service Error codes */
+#define BT_AICS_ERR_INVALID_COUNTER                0x80
+#define BT_AICS_ERR_OP_NOT_SUPPORTED               0x81
+#define BT_AICS_ERR_MUTE_DISABLED                  0x82
+#define BT_AICS_ERR_OUT_OF_RANGE                   0x83
+#define BT_AICS_ERR_GAIN_MODE_NO_SUPPORT           0x84
+
+/** @brief Opaque Audio Input Control Service instance. */
+struct bt_aics;
+
+/** @brief Structure for initializing a Audio Input Control Service instance. */
+struct bt_aics_init {
+	/** Initial audio input gain (-128 to 127) */
+	int8_t gain;
+
+	/** Initial audio input mute state */
+	uint8_t mute;
+
+	/** Initial audio input mode */
+	uint8_t mode;
+
+	/** Initial audio input gain units (N * 0.1 dB) */
+	uint8_t units;
+
+	/** Initial audio input minimum gain */
+	int8_t min_gain;
+
+	/** Initial audio input maximum gain */
+	int8_t max_gain;
+
+	/** @brief Initial audio input type */
+	int8_t input_type;
+
+	/** Initial audio input state (enabled/disabled) */
+	bool input_state;
```

Comment:
I believe this must be the "input status" from the spec (status, not state), and it can be either "inactive" or "active", not enabled/disabled.
I think it is clearer to follow the terminology from the spec.  Especially since the spec already has another characteristic called "input state"